sebastien15/capstoneproject
# Ft update an article

#### What does this PR do?

> This PR update an article on mongodb by looking for its id

#### Description of Task to be completed

>added route to access updateBlog
>added method to updateBlog

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `ft-update-an-artitcle`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/updateBlog/{id} in post man make sure it is a put method


#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/vWyMBqAc
#### Screenshots (if appropriate)
>![update article](https://user-images.githubusercontent.com/41580903/93981534-66c43f80-fd80-11ea-84fc-565a2eeac845.PNG)
